docs/configuration.md

```diff
@@ -1664,6 +1664,14 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.<particular protocol>.port</code></td>
```

Comment:
What does `<particular protocol>` mean here ? Is this config name parameterized ?